### DISCUSSION TITLE
Aria-live accessibility instruction display can't be disabled when the Select component is in focus

### DISCUSSION THREAD

**Discussion by Matthew-Mrakovcic** [JedWatson/react-select#5574] [Q&A]
<img width="380" alt="Screenshot 2023-03-03 111219" src="https://user-images.githubusercontent.com/98486825/222770515-0d60eef6-f178-4175-a3c2-6f15f6c02bd0.png">

Aria-live accessibility instruction display can't be disabled when the Select component is in focus. See image for reference. Has anyone come across this issue or have any ideas to resolve it? Thank you in advance!

**Comment by goldmont** [comment]
Hi,

same issue here. It's the first time that aria-live instructions appear to me. I don't understand why these instructions don't appear also in other websites I made. I cannot find a solution. I don't know if it's a bug.

In my last website:
![image](https://github.com/JedWatson/react-select/assets/6855692/c4e97b04-6bde-4f0b-990f-787a44aba5c2)

Another website using the same version of this library:
![image](https://github.com/JedWatson/react-select/assets/6855692/56503afb-bf28-40e3-83c7-fb57b784b816)
